From 83e70aef90d7320f3eda156a0c72dff2560ecefa Mon Sep 17 00:00:00 2001
From: xbw <xbw@rock-chips.com>
Date: Wed, 3 Jul 2013 13:15:33 +0800
Subject: [PATCH] rk2928: eliminate the build error using
 rk2928_tb_defconfig/rk2928_sdk_defconfig. as the following:   UPD    
 include/generated/compile.h   CC      init/version.o   LD     
 init/built-in.o   LD      .tmp_vmlinux1 drivers/built-in.o: In function
 rk29_sdmmc_gpio_open' make: *** [.tmp_vmlinux1] Error 1

---
 arch/arm/mach-rk2928/board-rk2928-sdk-sdmmc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/mach-rk2928/board-rk2928-sdk-sdmmc.c b/arch/arm/mach-rk2928/board-rk2928-sdk-sdmmc.c
index dfb3251c4314..3031a8ffd213 100755
--- a/arch/arm/mach-rk2928/board-rk2928-sdk-sdmmc.c
+++ b/arch/arm/mach-rk2928/board-rk2928-sdk-sdmmc.c
@@ -933,7 +933,8 @@ static struct rksdmmc_gpio_board rksdmmc1_gpio_init = {
 *************************************************************************/
 
 #if !defined(CONFIG_SDMMC_RK29_OLD)	
-static void rk29_sdmmc_gpio_open(int device_id, int on)
+//static void rk29_sdmmc_gpio_open(int device_id, int on)
+void rk29_sdmmc_gpio_open(int device_id, int on)
 {
     switch(device_id)
     {
-- 
2.35.3

